src/jdk.jpackage/windows/native/common/WinFileUtils.cpp

```diff
@@ -649,6 +652,12 @@ FileWriter& FileWriter::write(const void* buf, size_t bytes) {
     return *this;
 }
 
+FileWriter& FileWriter::writeln(const tstring& line) {
```

Comment:
How this function is used?

Reply:
yes - not used.
code left over from earlier instance where we wrote into a user cfg file to record where we found runtime.  Current implementation never writes files in native code, and this can be removed.